package.json

```diff
@@ -43,6 +43,8 @@
     "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch" : "grunt watch",
     "build": "grunt",
+    "lint": "grunt eslint",
```

Comment:
Can you file a follow up bug: our grunt eslint task should be checking all .js files and currently doesn't (my bad, probably). It looks like what you use for the fix command is more what we want.